src/Visualization/Python/GenerateVideo.py

```diff
@@ -0,0 +1,312 @@
+#!/usr/bin/env python
+
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+import click
+import glob
+import logging
+from paraview.simple import *
+
+#### disable automatic camera reset on 'Show'
+paraview.simple._DisableFirstRenderCameraReset()
+
+
+def generate_video(file_destination, volume_xmf, aha_xmf, ahb_xmf,
+                   output_destination, camera_angle):
+    """Generate Pictures from XMF files for BBH Visualizations
+
+    Generate pictures from BBH runs using the XMF files generated using
+    generate-xdmf. For files to be read properly, the XMF files being pointed to
+    should be in the same directory as the Volume.h5 and Surfaces.h5 files.
+
+    To splice all the pictures into a video, look into FFmpeg"""
+
+    # create a new 'XDMF Reader'
```

Comment:
Are these comments autogenerated by ParaViews recording feature? Either way, please remove comments that don't give any more information than the line of Python code (I don't need a comment explaining that the next line creates a new XDMF reader when the next line literally reads `reader = XDMFReader()`, or a comment saying "get animation scene" when the next line is `GetAnimationScene()`).

Please do keep or add comments that give some more context or explanations about the code, or that split the code into logical paragraphs.

Reply:
Yeah, most of the comments are auto generated, I only added where I thought it might be helpful, will remove most of these